Datadog/E2ETests/NTP/KronosE2ETests.swift

```diff
@@ -7,22 +7,45 @@
 @testable import Datadog
 
 class KronosE2ETests: E2ETests {
```

Comment:
It is to also collect this telemetry from our E2E test runs - more data, always good 💭.